src/styles/utils.scss/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%colorTransition {
+  transition: color 0.5s;
```

Comment:
during 300ms. 0.5s =500ms

Reply:
:harold:

Reply:
let's fit it
<img width="443" alt="image" src="https://user-images.githubusercontent.com/86183007/216611472-3f08e1ff-cb02-40c1-ad9d-99048992841c.png">